lib/clients/evPump/evPumpClient.py

```diff
@@ -0,0 +1,210 @@
+        self.setSizePolicy(QtGui.QSizePolicy.Minimum, QtGui.QSizePolicy.Fixed)
+        self.title.setFont(font)
+        self.title.setAlignment(QtCore.Qt.AlignCenter)
+        
+        self.templabel = QtGui.QLabel('Diode Temp degC')
+        self.currentlabel = QtGui.QLabel('Current')
+        self.powerlabel = QtGui.QLabel('Power')
+        self.powercontrollabel = QtGui.QLabel('Power Control')
+        self.statuslabel = QtGui.QLabel('Laser Status: ')
+        self.currentcontrollabel = QtGui.QLabel('Current Control (A)')
+        self.controlmodelabel = QtGui.QLabel('Control Mode')
+        
+        self.controlcombo = QtGui.QComboBox()
+        self.controlcombo.addItems(['Power Control','Current Control'])
+
+        self.powerspinbox = QtGui.QDoubleSpinBox()
+        self.powerspinbox.setFont(QtGui.QFont('MS Shell Dlg 2',pointSize=16))
+        self.powerspinbox.setDecimals(2)
+        self.powerspinbox.setSingleStep(0.01)
+        self.powerspinbox.setRange(0.0,15.0)
+        self.powerspinbox.valueChanged.connect(self.changePower)
+        self.powerspinbox.setKeyboardTracking(False)
+        
+        self.currentspinbox = QtGui.QDoubleSpinBox()
+        self.currentspinbox.setFont(QtGui.QFont('MS Shell Dlg 2',pointSize=16))
+        self.currentspinbox.setDecimals(3)
+        self.currentspinbox.setSingleStep(0.001)
+        self.currentspinbox.setRange(0.0,7.2)
+        self.currentspinbox.valueChanged.connect(self.changeCurrent)
+        self.currentspinbox.setKeyboardTracking(False)
+        
+        self.tempdisplay = QtGui.QLCDNumber()
+        self.tempdisplay.setDigitCount(5)
+
+        self.currentprogbar = QtGui.QProgressBar()
+        self.currentprogbar.setOrientation(QtCore.Qt.Vertical)
+        
+        self.laserswitch = QCustomSwitchChannel('Laser',('On','Off'))
+        self.shutterswitch = QCustomSwitchChannel('Shutter',('Open','Closed'))
+
+        self.powerprogbar = QtGui.QProgressBar()
+        self.powerprogbar.setOrientation(QtCore.Qt.Vertical)
+        self.powerprogbar.setMaximum(100)
+        self.powerprogbar.setMinimum(0)
+
+        layout.addWidget(self.title,          0,0,1,3)
+        layout.addWidget(self.currentprogbar, 2,0,8,1)
+        layout.addWidget(self.powerprogbar,   2,1,8,1)
+        layout.addWidget(self.currentlabel,   1,0,1,1)
+        layout.addWidget(self.powerlabel,     1,1,1,1)
+        layout.addWidget(self.templabel,      10,0,1,2)
+        layout.addWidget(self.statuslabel,    11,0,1,1)
+        layout.addWidget(self.tempdisplay,    9,2,4,1)
+        layout.addWidget(self.laserswitch,    6,2)
+        layout.addWidget(self.shutterswitch,    7,2)
+        layout.addWidget(self.powercontrollabel,   2,2) 
+        layout.addWidget(self.currentcontrollabel,   4,2)        
+        layout.addWidget(self.powerspinbox,       3,2)
+        layout.addWidget(self.currentspinbox,       5,2)
+        layout.addWidget(self.controlcombo,       1,2)
```

Comment:
actually for matrix readability I think its best to deviate from pep8 here. its essentially impossible to see where the widgets are if they dont line up